gradle/libs.versions.toml

```diff
@@ -14,29 +14,29 @@ minimessage = "4.12.0"
 luckperms = "5.4"
 guava = "31.1-jre"
 floodgate = "2.2.0-SNAPSHOT"
-kotlin = "1.8.0"
+kotlin = "1.8.10"
 event = "3.0.0"
 
 # Platforms
 bukkit = "1.16.5-R0.1-SNAPSHOT"
 bungeecord = "1.19-R0.1-SNAPSHOT"
-krypton = "a81b4baaed" # Krypton uses commit-based versioning
-velocity = "3.1.2-SNAPSHOT"
-sponge7 = "7.0.0"
-sponge8 = "8.0.0"
+krypton = "7f7a457cdf" # Krypton uses commit-based versioning
+velocity = "3.2.0-SNAPSHOT"
+sponge7 = "7.4.0"
+sponge8 = "8.1.0"
 
 # Platform extras
 placeholderapi = "2.11.2"
 vault = "1.7"
 via = "4.5.1"
 authlib = "1.5.25"
-bstats = "3.0.0"
+bstats = "3.0.1"
 redisBungee = "0.10.1"
 spark = "0.1-SNAPSHOT"
 
 # Plugins
-shadow = "7.1.2"
-lombok = "6.6.1"
+shadow = "8.1.0"
```

Comment:
The compilation error comes because the latest version of Shadow changed its groupId 

The line 
`plugin-shadow = { group = "gradle.plugin.com.github.johnrengelman", name = "shadow", version.ref = "shadow" }`
you must replace it with 
`plugin-shadow = { group = "com.github.johnrengelman", name = "shadow", version.ref = "shadow" }`

https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow

Reply:
yes ik that for the compilation error and reported it to johnrengelman as its not the groupid its the way libs work but that isn't the issue we need to fix its a issue with sponge